third_party/enophone/enophone_streaming.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import time
+import logging
+import random
+import numpy as np
+
+import pyqtgraph as pg
+from pyqtgraph.Qt import QtGui, QtCore
+
+import brainflow
+import enotools
+from brainflow.board_shim import BoardShim, BrainFlowInputParams, BoardIds, BrainFlowError
+from brainflow.data_filter import DataFilter, FilterTypes, AggOperations, WindowFunctions, DetrendOperations
+
+
+class Graph:
+    def __init__(self, board_shim):
+        self.board_id = board_shim.get_board_id()
+        self.board_shim = board_shim
+        self.exg_channels = BoardShim.get_exg_channels(self.board_id)
+        self.sampling_rate = BoardShim.get_sampling_rate(self.board_id)
+        self.update_speed_ms = 50
+        self.window_size = 2500
```

Comment:
for convenience lets use widnow size in seconds and add smth like `num_points = window_size * BoardShim.get_sampling_rate(BoardIds.ENOPHONE_BOARD)`